common-lib/src/main/scala/com/gu/mediaservice/lib/auth/Store.scala

```diff
@@ -66,6 +66,7 @@ class KeyStore(bucket: String, credentials: AWSCredentials) extends BaseStore[St
 object PermissionType extends Enumeration {
```

Comment:
![icon_offtopic](https://cloud.githubusercontent.com/assets/31692/8903850/c4428e66-3452-11e5-81e0-092423533f9e.gif)
What do we think of enumerations? That's what I was trying to avoid by using case objects and traits in the usage rights, which seems a [common](http://stackoverflow.com/questions/1898932/case-objects-vs-enumerations-in-scala) pattern (or even better things like [enumeratum](https://github.com/lloydmeta/enumeratum)). The basic task to try and get matching working.